project/Release.scala

```diff
@@ -27,14 +27,32 @@ object Release extends ParadoxKeys {
     val projectRef = extracted.get(thisProjectRef)
     val repo = extracted.get(Publish.defaultPublishTo)
     val state1 = extracted.runAggregated(publishSigned in projectRef, state)
-    // Make sure you set "-Dakka.genjavadoc.enabled=true" otherwise no
-    // japi will be generated and this following match will fail:
-    val (state2, Seq(japi, api)) = extracted.runTask(unidoc in Compile, state1)
-    val (state3, docs) = extracted.runTask(paradox in ProjectRef(projectRef.build, "akka-docs") in Compile, state2)
 
     IO.delete(release)
     IO.createDirectory(release)
     IO.copyDirectory(repo, release / "releases")
+    state1
+  }
+
+  def buildDocsCommand = Command.command("buildDocs") { state =>
+    if (!sys.props.contains("akka.genjavadoc.enabled"))
+      throw new RuntimeException("Make sure you start sbt with \"-J-Dakka.genjavadoc.enabled=true\" otherwise no japi will be generated")
+    val extracted = Project.extract(state)
+    // we want to build the api-docs and docs with 2.12.x
+    val scalaV = extracted.get(scalaVersion) 
+    if (!scalaV.startsWith("2.12"))
```

Comment:
is it possible to get to `crossScalaVersions.head` here instead?

Reply:
Do we know that will always be the version we want to build the docs with? I was thinking this is something we explicitly want to decide on.

Reply:
crossScalaVersions.head is what we use as default version so probably the one we want to use for docs

not very important though, since we are changing default version once every 2 years or so

Reply:
Sounds like we are relatively sure that we want to use the default version, I agree.